NetKAN/TextureReplacerReplaced.netkan

```diff
@@ -0,0 +1,17 @@
+{
+	"spec_version": "v1.4",
+	"name": "TextureReplacerReplaced",
+	"$kref": "#/ckan/github/HaArLiNsH/TextureReplacerReplaced",
+	"identifier": "TextureReplacerReplaced",
+	"release_status": "testing",
+	"ksp_version": "1.3.0",
```

Comment:
The release includes a version file, could you delete `ksp_version` and add `"$vref": "#/ckan/ksp-avc",`